bugnote_view_inc.php

```diff
@@ -199,7 +199,7 @@
 		?>
 		<div class="clearfix"></div>
 		<div class="space-2"></div>
-		<div class="btn-group-sm">
+		<div class="btn-group visible-on-hover">
```

Comment:
two reasons:
- other buttons in the pages are within plain "btn-group"
- to make it apply the margins with the defined css selector: `.btn-group .single-button-form `.
Otherwise, the selector should be repeated for every .btn-group-xx

What do you think?